modules/@apostrophecms/schema/ui/apos/logic/AposInputWrapper.js

```diff
@@ -67,6 +67,26 @@ export default {
 
       return label;
     },
+    // Build interpolation object when a translation key contains interpolations
+    // following this pattern: "aTranslationKey|foo=bar|fizz=buzz".
+    // i18n will be given the "aTranslationKey" key with following object:
+    // { foo: "bar", fizz: "buzz" }
+    //
+    // Usage example:
+    //
+    // ```
+    // help: `aposMultisite:shortNameHelp|baseDomain=${self.baseDomain}`
+    // ```
```

Comment:
@boutell what do you think? I tried to come up with a simple solution

Reply:
cf https://github.com/apostrophecms/multisite/pull/107/files#diff-514affe4dea6930297c29e391064e783ad8dabe940ed8a9f3b79369efb97b786R36